src/sentry/static/sentry/app/components/fileChange.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import Avatar from '../components/avatar';
+
+import TooltipMixin from '../mixins/tooltip';
+import ApiMixin from '../mixins/apiMixin';
+
+const FileChange = React.createClass({
+  propTypes: {
+    filename: React.PropTypes.string.isRequired,
+    authors: React.PropTypes.array.isRequired,
+    types: React.PropTypes.object.isRequired,
+  },
+
+  mixins: [
+    ApiMixin,
+    TooltipMixin({
+      selector: '.tip'
+    }),
+  ],
+
+  getInitialState() {
+    return {
+      loading: true,
+    };
+  },
+
+  render() {
+    let {filename, authors, types} = this.props;
+    authors = Array.from(authors);
```

Comment:
Why `Array.from`? AFAICT, these props are already arrays, and you don't seem to be mutating the list?